bg3-modders-multitool/bg3-modders-multitool/Models/StatStructures/CriticalHitTypeData.cs

```diff
@@ -0,0 +1,27 @@
+﻿/// <summary>
+/// Critical hit type data
+/// </summary>
+namespace bg3_modders_multitool.Models.StatStructures
+{
+    public class CriticalHitTypeData : StatStructure
+    {
+        public string AcidFx { get; set; }
```

Comment:
should this be `AcidFX`?

Reply:
> should this be `AcidFX`?

AcidFX yes, thanks for catching that. Its a typo.